shared/test/showcase/echo/chat_test.rb

```diff
@@ -32,7 +32,6 @@ def test_chat
 
     Thread.new do
       10.times do |n|
-        puts "PUSH hello #{n}:1"
```

Comment:
Can we add a conditional instead of removing these? I prefer to see them.

```ruby
puts "PUSH hello #{n}:1" if ENV["VERBOSE"]
```

Reply:
I don't mind leaving them if you actually want them. I assumed it was just an accident - they drive me crazy though 😄. I'll revert it.